app/controllers/dashboard_controller.rb

```diff
@@ -4,11 +4,11 @@ def show
     @role = params[:role]
 
     case @role
-    when 'Mentor'
+    when 'mentor'
       @mentors_data = CohortMember.where(role: 'Mentor')
-    when 'Mentee'
+    when 'mentee'
       @mentees_data = CohortMember.where(role: 'Mentee')
-    when 'Admin'
+    when 'admin'
```

Comment:
Thanks for changing these, I realized it didn't need to be capitalized later in the day😂